dist/server/setenv.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
What is this file for?

Reply:
forget to delete... 
I moved settings from setdist.sh to setenv.sh and deleted setdist.sh